edbdeploy/data/ansible/kubernetes_roles/remove_cloudnativepg_sandbox/README.md

```diff
@@ -0,0 +1,69 @@
+# remove_cloudnativepg_sandbox
```

Comment:
Duplicate role in edb-ansible:
https://github.com/EnterpriseDB/edb-ansible/tree/master/roles/remove_cloudnativepg_sandbox